### PR TITLE
Moving to common scripts part two

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5042

## Description
moving DBotPredictURLPhishing script from **Phishing URL** to **common scripts**.

## Minimum version of Cortex XSOAR
- [X] 6.0.0

## Does it break backward compatibility?
   - [X] Yes

### CODE REVIEW

File: Packs/CommonScripts/pack_metadata.json [hunk 1]
```diff
@@ -47,6 +47,14 @@
         "RemoteAccess": {
             "mandatory": false,
             "display_name": "RemoteAccess v2"
+        },
+        "rasterize": {
```
Comment (Shellyber) [inline Packs/CommonScripts/pack_metadata.json:51]: Why did we add these dependencies? 
It shouldn't be mandatory: true

**Review by Shellyber — COMMENTED** [review]
Looks good.
See my comment regarding the dependencies.
Also, add in the template the link to the issue

File: Packs/CommonScripts/pack_metadata.json (hunk 1 above)
Comment (maimorag) [reply Packs/CommonScripts/pack_metadata.json:51]: the script DBotPredictURLPhishing.py uses rasterize and Whois

**Comment by maimorag** [comment]
DBotPredictURLPhishing will stay in PhishingURL folder

**Pull request closed without merging**